### DISCUSSION TITLE
Monthly Meeting

### DISCUSSION THREAD

**Discussion by georgkrause** [OpenPodcastAPI/api-specs#15] [General / Meta]
Hello everyone, in our last meeting we discussed that we would like to do a monthly meeting from now on. We settled on

**second Tuesday; 3pm MET**

The Meeting takes place in our Matrix room https://matrix.to/#/!ZHdcrdWSgxXRREuJdU:matrix.org?via=matrix.org&via=tchncs.de&via=fachschaften.org

Next date:  30.5. 9pm MET

**Comment by georgkrause** [comment]
The next meeting will take place at 30.5. 9pm MET

**Reply by keunes** [reply]
That'd be 20:00 MET if I understand correctly, no? Given that we're meeting 21:00 CEST.
